app/views/collections/_collections.html.slim

```diff
@@ -16,29 +16,14 @@
         span= collection.title
     .card-body
       .row.px-3
-        .col
-          .description.collapsable.w-100
-            - if collection.description.blank?
-              span style=("color: gray")
-                = t('collections.shared.no_description')
-            - else
-              = render_markdown(collection.description)
-          .more-btn-wrapper
-            = button_to collection, class: 'more-btn' do
-              .more-tag
-                = t('common.button.show_more')
-                span.fa-solid.fa-caret-down.ms-1
-              .less-tag.d-none
-                = t('common.button.show_less')
-                span.fa-solid.fa-caret-up.ms-1
-          br
```

Comment:
Aren't you removing the collection description here, adding in the previous PR #1124? To me, this seems like a mistake caused by Git...